ocaml_autodoc/dune

```diff
@@ -0,0 +1,7 @@
+(executable
```

Comment:
No more `jbuild_version` in `dune` files : [OK, it's been deprecated](https://dune.readthedocs.io/en/latest/dune-files.html#jbuild-version)